frontend/src/components/Shared/Authcommon/FormElements.jsx

```
@@ -115,30 +149,46 @@ PasswordInput.propTypes = {
-    <span className="absolute inset-0 w-full h-full bg-gradient-to-r from-blue-500 to-blue-700 rounded-[14px] shadow-md" />
-    <span className="absolute inset-0 w-full h-full bg-white/10 rounded-[14px] blur-[1px]" />
-    <span className="absolute inset-0 w-full h-full bg-blue-600 rounded-[14px] transform transition-transform group-hover:scale-[1.02]" />
-    <span className="relative flex items-center justify-center text-white font-medium py-2.5 text-sm">
-      {loading ? loadingText : text}
+    <span className="absolute inset-0 w-full h-full bg-gradient-to-r from-blue-500 to-blue-700 rounded-[12px] sm:rounded-[14px] shadow-md" />
+    <span className="absolute inset-0 w-full h-full bg-white/10 rounded-[12px] sm:rounded-[14px] blur-[1px]" />
+    <span className="absolute inset-0 w-full h-full bg-blue-600 rounded-[12px] sm:rounded-[14px] transform transition-transform group-hover:scale-[1.02] group-hover:brightness-110" />
+    <span className="relative flex items-center justify-center text-white font-medium py-2 sm:py-2.5 text-sm">
+      {loading ? (
+        <div className="flex items-center">
+          <svg className="animate-spin -ml-1 mr-2 h-4 w-4 text-white" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)